projects/stargazer/plugins/other/radius/server.h

```diff
@@ -3,7 +3,10 @@
 #include "radproto/socket.h"
 #include "radproto/packet.h"
 #include "radproto/dictionaries.h"
+#include "stg/logger.h"
 #include <boost/asio.hpp>
+#include <stop_token>
```

Comment:
It should be `stop_token.hpp`. We use a 3rd-party library for `std::jthread` and `std::stop_token` because compilers may not have them yet.